### ISSUE TITLE
Introduce a new Calculation Mode pick-list to reflect "Schedule Calculate" scenarios

### ISSUE THREAD

**Issue by afawcett** [afawcett/declarative-lookup-rollup-summaries#392]
Users that want to use this facility, get confused about which Calculation Mode to pick, typically the Process Builder choice is the best, but this is far from obvious. More confusing is that the picklist already has a Scheduled entry in it (which is quite different from Schedule Calculate). Maybe have "Scheduled Full" ?

**Comment by jhoosemans** [comment]
Yes I agree! I'm confused right now about the Calculation Mode. I have created two Rollups that need once a week recalculation. I scheduled them, but what mode do I need to select now? I thought PB was a good one, but started doubting because it's actually a scheduled Rollup but they should NOT be included in the general Scheduled Rolups.

**Comment by afawcett** [comment]
If you used the 'Schedule Calculate' button for a full recalc each schedule, just set Calculation Mode to PB and you don't need to deploy a trigger either.

If you want scheduled recalc only of parent records the tool determines need to be recalculated, set Calculation Mode to Scheduled and use the Apex Classes page to scheduled the general rollup scheduler. This will use records stored in the scheduled line items object to determine what needs to be recalculated. 

The later option is lighter on the system and will run faster each schedule. However like realtime, the logic to detect whats changed and thus what parents need recalc is flawed by time based formula functions in your criteria. Thus full recalc in these situations is better. 

Hope this helps.
